src/screens/Drawer/MyDrawer.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import Onboarding from '../Onboarding/Onboarding';
+import { StyleSheet, Image, SafeAreaView, Text, View, ViewStyle, TextStyle } from 'react-native';
+import {
+    createDrawerNavigator, DrawerNavigationOptions,
+  } from '@react-navigation/drawer';
+  import { SIZES, COLORS, FONTS } from '../../styles/theme';
+
+import RampScreen from '../Ramp/Ramp';
+import SettingsScreen from '../Settings/Settings';
+import HelpScreen from '../Help/Help';
+import { GovernanceScreen } from '../Govenance/Governace';
+import OpenRequestsScreen from '../TransactionRequests/OpenRequestsScreen';
+import PendingRequestsScreen from '../TransactionRequests/PendingRequestsScreen';
+import TransactionHistoryScreen from '../TransactionRequests/TransactionHistoryScreen';
+import CustomDrawerContent from './CustomDrawer';
+import { createStackNavigator } from "@react-navigation/stack";
+
+
+const Stack = createStackNavigator();
+
+function StackNav() {
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name="Home" component={Onboarding} />
+      <Stack.Screen name="PendingRequests" component={PendingRequestsScreen} />
+      <Stack.Screen name="TransactionHistory" component={TransactionHistoryScreen} />
+      <Stack.Screen name="Governance" component={GovernanceScreen} />
+      <Stack.Screen name="Tamp" component={RampScreen} />
+      <Stack.Screen name="Help" component={HelpScreen} />
+      <Stack.Screen name="Settings" component={SettingsScreen} />
+      <Stack.Screen name="OpenRequests" component={OpenRequestsScreen} />
+    </Stack.Navigator>
+  );
+}
+
+const Drawer = createDrawerNavigator();
+
+export default function MyDrawer() {
+  return (
+    <Drawer.Navigator
+      screenOptions={{
+        headerShown: false,
+        drawerStyle: {
+          width: SIZES.width * 0.8,
+          borderRadius: 15,
+          backgroundColor: '#F5F5F5',
```

Comment:
@Chegemartinx @Allen-Muhani there is a background color her. That one should be in a central place how do we do that?

Reply:
Done